interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/PublisherToStreamTest.scala

```diff
@@ -0,0 +1,42 @@
+package scalaz.zio.interop.reactiveStreams
+
+import org.scalatest.{ Matchers, WordSpec }
+import scalaz.zio.Exit.Cause
+import scalaz.zio.stream.{ Sink, Stream }
+import scalaz.zio.{ DefaultRuntime, Exit }
+
+class PublisherToStreamTest extends WordSpec with Matchers with DefaultRuntime {
+
+  "A PublisherStream" should {
```

Comment:
Can we keep the non-TCK tests on specs2 and use the existing `TestRuntime` base trait?

Reply:
done